Microsoft.Toolkit.Uwp.UI.Controls/BladeControl/BladeControl.cs

```diff
@@ -67,6 +70,11 @@ protected override void OnApplyTemplate()
         {
             base.OnApplyTemplate();
             CycleBlades();
+
+            if (FullScreenBlades)
```

Comment:
Move the if check into the `HandleFullScreenBlades` method instead of checking before calling it each time.